main.go

```diff
@@ -98,15 +98,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	pcfg, err := sharedConfig.LoadFrom(configFile)
+	cfg, err := sharedConfig.Config(configFile)
 	if err != nil {
-		setupLog.Error(err, "failed to load the configuration file")
-		os.Exit(1)
-	}
-
-	cfg, err := pcfg.Inflate()
-	if err != nil {
-		setupLog.Error(err, "failed to initialize the configuration file")
+		setupLog.Error(err, "Failed to load the configuration")
```

Comment:
Yes, the previous error message was stupid, but see? You're using "load" in the (much better) error message for a method called `Config` which supports my previous point about the rename from `LoadFrom` to `Config` 😉